features/gba-dev/onbekend_waardes_gba.feature

```diff
@@ -0,0 +1,563 @@
+    | 'kind'    | kinderen      | geboorte                    | land   | geboorteland (03.30)                                                | 0000   |
+
+  Scenario: onbekend waarde voor nationaliteit
+    Gegeven de persoon met burgerservicenummer '000000280' heeft een 'nationaliteit' met de volgende gegevens
+    | naam                            | waarde   |
+    | nationaliteit (05.10)           | 0000     |
+    | reden opname (63.10)            | 311      |
+    | datum ingang geldigheid (85.10) | 20030417 |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 000000280                           |
+    | fields              | burgerservicenummer,nationaliteiten |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 000000280 |
+    En heeft de persoon een 'nationaliteit' met de volgende gegevens
+    | naam                              | waarde                               |
+    | nationaliteit.code                | 0000                                 |
+    | nationaliteit.omschrijving        | Onbekend                             |
+    | redenOpname.code                  | 311                                  |
+    | redenOpname.omschrijving          | Vaststelling onbekende nationaliteit |
+    | datumIngangGeldigheid             | 20030417                             |
+
+  Scenario: onbekend waarde voor reden opname nationaliteit
+    Gegeven de persoon met burgerservicenummer '000000280' heeft een 'nationaliteit' met de volgende gegevens
+    | naam                            | waarde   |
+    | nationaliteit (05.10)           | 0052     |
+    | reden opname (63.10)            | 000      |
+    | datum ingang geldigheid (85.10) | 20030417 |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 000000280                           |
+    | fields              | burgerservicenummer,nationaliteiten |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 000000280 |
+    En heeft de persoon een 'nationaliteit' met de volgende gegevens
+    | naam                              | waarde    |
+    | nationaliteit.code                | 0052      |
+    | nationaliteit.omschrijving        | Belgische |
+    | redenOpname.code                  | 000       |
+    | redenOpname.omschrijving          | Onbekend  |
+    | datumIngangGeldigheid             | 20030417  |
+
+Rule: de onbekend waarde voor geslacht wordt wel geleverd
```

Comment:
Als de Rule in regel 127 niet geldt in de GBA API situatie maar de door mij voorgestelde Rule dan hoeft deze regel niet afzonderlijk vermeldt te worden.

Reply:
Ik heb het woordje wel weggehaald. Ik vermoed dat deze straks opgedeeld gaat worden en dan kan dit scenario in de map "geslacht" worden ondergebracht. Ik ga hem nu niet integreren.